File: BLL/Objects/TimeEntry.cs
using System;
using System.Runtime.Serialization;

namespace PokerTracker.BLL.Objects
{
    [DataContract]
    public class TimeEntry : IdObject
    {
        [DataMember]
        public DateTime RecordedAt { get; set; }

        [DataMember]
        public int StackSize { get; set; }

        [DataMember]
        public int DealerTokes { get; set; }

        [DataMember]
        public int ServerTips { get; set; }
    }
}

File: Web/js/services/TimeEntry.js
angular.module('pokerTracker.services', [])
    .factory('TimeEntry', [function () {
        return function TimeEntry() {
            this.RecordedAt = null;
            this.StackSize = null;
            this.DealerTokens = null;
            this.ServerTips = null;
        };
    }
]);